### PR TITLE
Feature/oe 252 checkbox to sum numbers selecteds

### PR DESCRIPTION
**Pull request by Ron-Junior**
## Goal

Add check box to sum numbers

## Review

For the submitter, initial self-review:

- [ ] Reviewed the test cases added for completeness and possible points for discussion
- [ ] Check the scope of the changeset - is everything in the diff required for the pull request?

For the pull request reviewer(s), this changeset has been reviewed for:

- [ ] Consistency between the changeset and the goal stated above
- [ ] Usage friction - is the proposed change in usage cumbersome or complicated?
- [ ] Complexity - is the proposed change introducing any unjustified complexity? 
- [ ] Thoroughness of added tests and any missing edge cases

### CODE REVIEW

File: app/Http/Livewire/Components/NumberTrackerDetailAccordionTable.php [hunk 1]
```diff
@@ -64,4 +73,85 @@ public function collapseOffice( int $regionIndex, int $officeIndex)
...
+            return $office;
+        });
+    }
+
+    public function collectUsers($office)
+    {
+        return collect($office->users)->map(function ($user) {
+            return $user;
+        });
+    }
```
Comment (devsquad-mathews-lima) [inline app/Http/Livewire/Components/NumberTrackerDetailAccordionTable.php:138]: Maybe you can replace these functions with something like

```
    private function getCollectionOf($array)
    {
        return collect($array)->map(fn ($value) => $value);
    }
```

and use this way

```
$collectOffice = $this->getCollectionOf($region->offices);
$office->users = $this->getCollectionOf($office->users);
```

_it's just an ideia_

**Review by devsquad-mathews-lima — CHANGES_REQUESTED** [review]

**Review by devsquad-mathews-lima — APPROVED** [review]
